src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -424,7 +424,6 @@ private ClaimsIdentity CreateClaimsIdentity(JsonWebToken jwtToken, TokenValidati
         /// <exception cref="ArgumentNullException">if 'validationParameters' is null.</exception>
         /// <exception cref="SecurityTokenException">if 'jwtToken.Enc' is null or empty.</exception>
         /// <exception cref="SecurityTokenDecompressionFailedException">if decompression failed.</exception>
-        /// <exception cref="SecurityTokenEncryptionKeyNotFoundException">if 'jwtToken.Kid' is not null AND decryption fails.</exception>
```

Comment:
We should get rid of SecurityTokenEncryptionKeyNotFoundException mention here as well:

https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/93cdb72619325df2d86c3a8afb79d22348191d7a/src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs#L1353

Reply:
Good catch. This reference has been removed.